src/modules/lodash.js

```diff
@@ -0,0 +1,12 @@
+import _ from 'lodash';
+
+export function component() {
+  const element = document.createElement('div');
+
+  // Lodash, now imported by this script
+  element.innerHTML = _;
+
+  return element;
+}
+
+export default component;
```

Comment:
- [ ] [OPTIONAL] you can remove this file if you are not using lodash anywhere. This file simply comes from the webpack tutorial so it's not necessary to have it if you are not using it.
- [ ] [OPTIONAL] you can remove the `lodash` dependency from your package.json as well